### ISSUE TITLE
Restore logging exceptions

### ISSUE THREAD

**Issue by fgregg** [mpastell/Pweave#111]
In v0.25, if an exception was encountered in a chunk, a message would be logged to stderr. This was removed at around https://github.com/mpastell/Pweave/commit/a4f5103a428c937a93914f22980890d42a3b93cd

@mpastell, would you be amenable to a PR that restored this type of logging? Probably using the Python's standard library logging module?

**Comment by kylebarron** [comment]
I agree with @fgregg; there has to be some sort of logging to console when Python raises exceptions. I just spent an hour trying to figure out why a document wasn't weaving correctly. I had `results` set to `hidden`, and so I got no notification ever that a file path was wrong in the code. More generally, when `results='hidden'` and logging is off, there's no way to know if portions of the file didn't run correctly.

I personally would advocate for a default where the document doesn't build unless all code chunks run without exceptions, with a command-line switch to turn off this behavior with loggin.
